src/element/builders.rs

```diff
@@ -345,6 +345,29 @@ macro_rules! ion_struct {
     }};
 }
 
+/// Constructs a [`Sequence`] with the specified child values.
+///
+/// Note that a `Sequence` is NOT a type of `Element`. However, one can convert a `Sequence` into a
+/// `List` or `SExp`.
+///
+/// ```
+/// use ion_rs::element::{Element, Sequence};
+/// use ion_rs::{ion_seq, ion_list};
+/// // Construct a Sequence from Rust values
+/// let actual: Sequence = ion_seq!["foo" 7 false ion_list![1.5f64, -8.25f64]];
+/// // Construct a Sequence from serialized Ion data
+/// let expected: Sequence = Element::read_all(r#" "foo" 7 false [1.5e0, -8.25e0] "#).unwrap();
```

Comment:
🗺️ `ion_seq!` doesn't take commas between its values in order to better approximate the syntax for top-level values in an Ion stream.

Reply:
Which approach does the `ion_sexp!` macro take? Do you think the similarity to Ion syntax is really that significant vs the familiarity of having a comma-delimited list of values?